helper/xen_orchestra_helper/xen_orchestra_helper.py

```diff
@@ -0,0 +1,43 @@
+from abc import ABC, abstractmethod
+class XenOrchestraHelper(ABC):
+
+    def _get_add_host_command(self, label, host, username, password):
```

Comment:
This doesn't need to be a private method. This can be public method.

Reply:
Changes made in latest commits